CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+ * Pull requests welcome!
+ * Pull requests should come with good tests
+ * All PRs should be backed by a JIRA ticket and included in the subject using the format:
+   * `Fixes 23: Some great feature` - if a PR fully resolves an issue or feature, or is the last PR in a series of PRs.
+   * `Refs 23: Partial work for issue` - if a PR partially resolves an issue or feature or is not the last PR in a series of PRs.
+   * If you do not have access to the Jira (we are working on opening it up), open a PR without this
+     and we will add it for you.
```

Comment:
```suggestion
   * If you do not have access to the Jira, open a PR without this and we will add it for you.
```

The jira is now public so its as open as it can be.  I think anyone can open a task, but i'm not 100% sure